mesos/src/main/java/brooklyn/entity/mesos/task/marathon/MarathonTaskImpl.java

```diff
@@ -493,7 +493,7 @@ public void stop() {
             // Docker command or args
             String command = config().get(COMMAND);
             builder.putIfNotNull("command", command);
-            List<String> args = config().get(ARGS);
+            List<String> args = MutableList.copyOf(config().get(ARGS));
```

Comment:
is it not possible to have other args for a marathon task other than an image entrypoint?